paramak/parametric_components/blanket_constant_thickness_arc_h.py

```diff
@@ -8,31 +9,29 @@ class BlanketConstantThicknessArcH(RotateMixedShape):
     horizontally for the thickness of the blanket to back of the blanket.
 
     Arguments:
-        inner_mid_point ((float, float)): the x,z coordinates of the mid
-            point on the inner surface of the blanket.
-        inner_upper_point ((float, float)): the x,z coordinates of the upper
-            point on the inner surface of the blanket.
-        inner_lower_point ((float, float)): the x,z coordinates of the lower
-            point on the inner surface of the blanket.
-        thickness (float): the radial thickness of the blanket in cm.
-        stp_filename (str, optional): Defaults to
-            "BlanketConstantThicknessArcH.stp".
-        stl_filename (str, optional): Defaults to
-            "BlanketConstantThicknessArcH.stl".
-        material_tag (str, optional): Defaults to "blanket_mat".
+        inner_mid_point: the x,z coordinates of the mid point on the inner
+            surface of the blanket.
+        inner_upper_point: the x,z coordinates of the upper point on the
+            inner surface of the blanket.
+        inner_lower_point: the x,z coordinates of the lower point on the
+            inner surface of the blanket.
+        thickness: the radial thickness of the blanket in cm.
+        stp_filename: Defaults to "BlanketConstantThicknessArcH.stp".
+        stl_filename: Defaults to "BlanketConstantThicknessArcH.stl".
+        material_tag: Defaults to "blanket_mat".
     """
 
     def __init__(
         self,
-        inner_mid_point,
-        inner_upper_point,
-        inner_lower_point,
-        thickness,
-        stp_filename="BlanketConstantThicknessArcH.stp",
-        stl_filename="BlanketConstantThicknessArcH.stl",
-        material_tag="blanket_mat",
+        inner_mid_point: Tuple[float, float],
+        inner_upper_point: Tuple[float, float],
+        inner_lower_point: Tuple[float, float],
+        thickness: float,
+        stp_filename: Optional[str] = "BlanketConstantThicknessArcH.stp",
+        stl_filename: Optional[str] = "BlanketConstantThicknessArcH.stl",
+        material_tag: Optional[str] = "blanket_mat",
         **kwargs
-    ):
+    ) -> None:
```

Comment:
Just a question: is this needed for every init method ?

Reply:
This indicates to the type hinting what type the class is when it is initialized. I guess this is wrong because the class doesn't return a None type object. It can be removed